### PR TITLE
Fix typos in the README

### PR DESCRIPTION
**Pull request by aknuds1**
Fix various typos in the README and fix a reference to bankai CLI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -387,7 +387,7 @@ overhead during runtime, so for production environments we should unwrap this
 using [yo-yoify][yo-yoify].
 
 Setting up browserify transforms can sometimes be a bit of hassle; to make this
-more convenient we recommend using [bankai][bankai] with `--optimize` to
+more convenient we recommend using [bankai][bankai] with `--watch=false` to
```
Comment (goto-bus-stop) [inline README.md]: hmm, this flag isn't supposed to exist; we should mention the `bankai build` command instead

Comment (aknuds1) [reply README.md]: @goto-bus-stop You mean it should read `bankai build` instead of just `bankai`?

Comment (aknuds1) [reply README.md]: @goto-bus-stop Or do you mean that `bankai build` will optimize without `--watch=false`? It's unclear to me.

Comment (goto-bus-stop) [reply README.md]: the latter!

Comment (aknuds1) [reply README.md]: @goto-bus-stop Gotcha, will update.

**Comment by aknuds1** [comment]
Thanks @goto-bus-stop! 🎉

**Pull request merged**